rust/sleeper_core/src/datafusion/filter_aggregation_config.rs

```diff
@@ -25,7 +25,7 @@ use datafusion::{
     common::{Column, DFSchema, HashSet, plan_datafusion_err, plan_err},
     dataframe::DataFrame,
     error::{DataFusionError, Result},
-    logical_expr::{AggregateUDF, Expr, ExprSchemable, ScalarUDF, col},
+    logical_expr::{AggregateUDF, Expr, ExprSchemable, ScalarUDF, ident},
```

Comment:
It looks like this change stops it normalising the column names? It looks like that means it was converting them to lower case before. Was this a bug? Where is it tested?